kyo-core/shared/src/main/scala/kyo/Log.scala

```diff
@@ -37,89 +91,82 @@ object Log extends LogPlatformSpecific:
         def warn(msg: => String, t: => Throwable)(using frame: Frame, allow: AllowUnsafe): Unit
         def error(msg: => String)(using frame: Frame, allow: AllowUnsafe): Unit
         def error(msg: => String, t: => Throwable)(using frame: Frame, allow: AllowUnsafe): Unit
+
+        def safe: Log = Log(this)
```

Comment:
Not sure I see the value in this to be honest. It's unlike other designs where the APIs are hidden behind an opaque type.

Reply:
The main reason for adding it was so it's possible to have `Log.get` and `Local.use` methods. It'd be strange to add them with a `Log.Unsafe` type. That's something useful because computations can get the `Log` instance once and avoid further `Local` lookups. Another use case is passing the instance to some unsafe execution context like library integrations. While not essential, adding it keeps the consistency with similar APIs and provides these benefits

Reply:
Can the local store the Safe variant?

Reply:
Yes, I've changed it to use safe variant